test/Intl/DateTimeFormatOptions.js

```diff
@@ -51,7 +51,7 @@ var tests = [
                 }
                 catch (e) {
                     if (!(e instanceof RangeError)) {
-                        fail("Incorrect exception was thrown.");
+                        assert.fail("Incorrect exception was thrown.");
```

Comment:
@veler I guess this is the issue you were talking about? Good catch. I guess we didn't hit this path before so nothing complained that the `fail` function was missing. What behavior changed that exposed this path for you?